Maestro/src/Microsoft.DotNet.Maestro/Config.cs

```diff
@@ -15,10 +18,10 @@ public class Config
         public static Config Instance { get; } = new Config();
 
         private Lazy<string> _webhookSecretToken = new Lazy<string>(() => ConfigurationManager.AppSettings[nameof(WebhookSecretToken)]);
-        private Lazy<string> _vsoUser = new Lazy<string>(() => ConfigurationManager.AppSettings[nameof(VSoUser)]);
-        private Lazy<string> _vsoPassword = new Lazy<string>(() => ConfigurationManager.AppSettings[nameof(VSoPassword)]);
         private Lazy<string> _subscriptionsUrl = new Lazy<string>(() => ConfigurationManager.AppSettings[nameof(SubscriptionsUrl)]);
         private Lazy<string> _azureWebJobsStorage = new Lazy<string>(() => ConfigurationManager.ConnectionStrings[nameof(AzureWebJobsStorage)]?.ConnectionString);
+        private Lazy<string> _vstsInstanceUserName = new Lazy<string>(() => ConfigurationManager.AppSettings[nameof(VstsUsername)]);
+        private Dictionary<string, string> _vstsCreds = new Dictionary<string, string>();
```

Comment:
Are instance names case sensitive?

Reply:
In our subscriptions.json they are all lower case.  Maybe worth changing to a case insensitive comparer though.  Are Configuration Manager keys insensitive?

Reply:
I don't know off hand but my guess is it's case insensitive. 

---
In reply to: [190612202](https://github.com/dotnet/versions/pull/296#discussion_r190612202) [](ancestors = 190612202)